app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,16 @@
+<% google_analytics_code = Tenant.find_by(database_name: Apartment::Tenant.current_tenant)&.google_analytics_code %>
```

Comment:
Hey @sarastanton -- just occurred to me that it might be wise to fall back to an existing .env var in cases where there's no value in the db.